### ISSUE TITLE
better support for raw text input (multiple sentences, parentheses, etc.)

### ISSUE THREAD

**Issue by jonorthwash** [jonorthwash/ud-annotatrix#123]
The [new logic for catching SD format](https://github.com/jonorthwash/ud-annotatrix/blob/master/standalone/lib/annotator.js#L452) fails on sentences with "("s in them, and doesn't catch plain text that's more than one line long.

The [conversion of text to CoNLL-U](https://github.com/jonorthwash/ud-annotatrix/blob/master/standalone/lib/converters.js#L43), though, does seem to support multi-line raw-text sentences.

**Comment by ftyers** [comment]
Can you give some examples of sentences with ( ) in that you would like to support, otherwise the bug is a bit vague :D

**Comment by jonorthwash** [comment]
If you type/paste in the sentence "This is a test (of syntax parsing).", it detects it as SD instead of plain text, but then fails to convert/tokenise it right.

**Comment by ftyers** [comment]
Now it detects that as plain text, see commit 53fa690. But still detects:

```
This is a test .
root(test)
```

and

```
This is a test . 
nsubj(test, This)
```
as SDParse. Can we close this issue ?

**Comment by jonorthwash** [comment]
Okay, I guess the parentheses issue is solved (for now), but #117 is still a problem.
